internal/storage/serde.go

```diff
@@ -166,6 +171,9 @@ var serdeMap = func() map[schemapb.DataType]serdeEntry {
 		func(any) uint64 {
 			return 1
 		},
+		func() parquet.Encoding {
+			return parquet.Encodings.Plain
```

Comment:
Per my understanding, since INT8 is stored as INT32 in parquet, using dictionary is always better than plain encoding.